marchenko/homework1

```diff
@@ -0,0 +1,3 @@
+:)
+hw is not ready sorry
```

Comment:
Your english very bad. Please, redo this.

Reply:
Ok. Done.